api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -264,6 +267,22 @@ public void alertUserRegisteredTierExpiration(final DbUser user, Instant expirat
         htmlMessage);
   }
 
+  @Override
+  public void alertUsersUnusedDiskWarningThreshold(
+      List<DbUser> users,
+      DbWorkspace diskWorkspace,
+      LeonardoListPersistentDiskResponse disk,
+      int daysUnused,
+      @Nullable Double workspaceInitialCreditsRemaining)
+      throws MessagingException {
+    log.info(
```

Comment:
This will not be merged, part 2/2 (which replaces this) will be stacked and merged to main at the same time as this PR.